docs/csharp/language-reference/keywords/in-parameter-modifier.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: "in parameter modifier - C# Reference"
 ms.custom: seodec18
-ms.date: 02/12/2019
+ms.date: 03/26/2019
 helpviewer_keywords: 
   - "parameters [C#], in"
   - "in parameters [C#]"
 ---
 
 # in parameter modifier (C# Reference)
 
-The `in` keyword causes arguments to be passed by reference. It is like the [ref](ref.md) or [out](out-parameter-modifier.md) keywords, except that `in` arguments cannot be modified by the called method. Whereas `ref` arguments may be modified,  `out` arguments must be modified by the called method, and those modifications are observable in the calling context.
+The `in` keyword makes the formal parameter an alias for the argument, which must be a variable. In other words, any operation on the parameter is made on the argument. It is like the [ref](ref.md) or [out](out-parameter-modifier.md) keywords, except that `in` arguments cannot be modified by the called method. Whereas `ref` arguments may be modified, `out` arguments must be modified by the called method, and those modifications are observable in the calling context.
```

Comment:
I'm not sure that I would remove the original first sentence -- "causes arguments to be passed by reference" is a very clear definition for many (or perhaps just me?). In contrast, I had to spend a few minutes thinking about what the substituted first sentence means. A really good solution seems to me to keep the original sentence, followed by "It makes the formal parameter an alias...."

Reply:
I'll note that this change was prompted by Eric Lippert in the initial #10116 issue.

> (1) We start with "passed by reference", but that is describing an implementation mechanism, and moreover, a mechanism likely not familiar to the novice. Better to describe the semantics: out, ref and in parameters make the formal parameter an alias for the argument, which must be a variable. That explains the feature at the level at which it is used, not at the level at which it is implemented.

Encouraging novices to mentally model these keywords as "alias" is ideal, but many programmers new to C# (but not to coding) will be far more familiar with the phrase, "passed by reference." Someone with absolutely no experience would likely be better served with the alias phrasing...but I'm it's appropriate to target such users; this isn't really a feature they should be using.